liquibase-service/src/main/resources/db/rdb/036-create_data_exchange_config-001.sql

```diff
@@ -0,0 +1,29 @@
+IF
+NOT EXISTS(SELECT *
+              FROM sys.databases
+              WHERE name = 'RDB')
+BEGIN
```

Comment:
I am not sure if we need an RDB table creation script because we're currently not managing RDB creation.

Reply:
It might be best to keep only the data_sync_config table as part of the create script.

Reply:
+1 we target rdb_modern and we can remove this piece .

Reply:
Lines 1 to 12 can be removed.